src/Microsoft.PowerShell.Commands.Management/resources/CmdletizationResources.resx

```diff
@@ -124,10 +124,6 @@
 {1} is a placeholder for a server name.  Example: "localhost".</comment>
   </data>
   <assembly alias="System.Windows.Forms" name="System.Windows.Forms, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b77a5c561934e089" />
```

Comment:
I thought this file can be removed, not?

Reply:
no, it's used and a different resource.